amethyst_renderer/src/pass/shaders/vertex/skybox.glsl

```diff
@@ -0,0 +1,31 @@
+// TODO: Needs documentation.
+
+#version 150 core
+
+layout (std140) uniform VertexArgs {
+    uniform mat4 proj;
+    uniform mat4 view;
+    uniform mat4 model;
+};
+
+in vec3 position;
+in vec3 normal;
+in vec2 tex_coord;
+
+out VertexData {
+    vec3 position;
+    vec3 normal;
+    vec2 tex_coord;
+} vertex;
+
+void main() {
+    mat4 view_without_translation = view;
+    view_without_translation[3][0] = 0.0f;
```

Comment:
`view_without_translation[3].xyz = vec3(0.0f, 0.0f, 0.0f);` maybe ?

Reply:
:+1: I continue to impress myself with my lack of swizzling